src/MetaMorpho.sol

```diff
@@ -513,7 +514,8 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
 
     /// @notice Accepts the pending cap of the market defined by `id`.
     function acceptCap(Id id) external afterTimelock(pendingCap[id].validAt) {
-        _setCap(id, pendingCap[id].value);
+        // Safe "unchecked" cast because pendingCap <= type(uint184).max.
```

Comment:
We could go for uint184 everywhere so that: we are only using `toUint184`, and we don't need to verify this property to make sure that this cast is safe. According to my tests it does not even cost more gas